pkg/apis/node/types.go

```diff
@@ -45,6 +46,19 @@ type RuntimeClass struct {
 	// The Handler must conform to the DNS Label (RFC 1123) requirements, and is
 	// immutable.
 	Handler string
+
+	// Overhead represents the resource overhead associated with running a pod for a
+	// given RuntimeClass. For more details, see
+	// https://git.k8s.io/enhancements/keps/sig-node/20190226-pod-overhead.md
```

Comment:
This should have a comment about being an alpha field, the same as the one on the pod field.